modules/godot/nodes/ecs_world.cpp

```diff
@@ -432,6 +432,7 @@ TypedArray<String> WorldECS::get_configuration_warnings() const {
 		warnings.push_back(TTR("The selected pipeline `") + active_pipeline + TTR("` doesn't exists in this WorldECS."));
 	}
 
+#ifdef TOOLS_ENABLED
```

Comment:
I wonder if wrap the entire function with `ifdef` would make more sense instead, what do you think?